scripts/site/_site/doc/app/share/codebox/codebox.component.html

```diff
@@ -15,11 +15,9 @@
   </section>
   <section class="code-box-meta markdown">
     <div class="code-box-title">
-      <a (click)="navigateToFragment()"
-      >{{ nzTitle }}
-        <a class="edit-button" [attr.href]="nzHref" target="_blank" rel="noopener noreferrer">
-          <span nz-icon nzType="edit"></span>
-        </a>
+      <a (click)="navigateToFragment()">{{ nzTitle }}</a>
+      <a class="edit-button" [attr.href]="nzHref" target="_blank" rel="noopener noreferrer">
+        <span nz-icon nzType="edit"></span>
       </a>
```

Comment:
Browsers flatten nested anchor elements, which can cause the nodes to be rendered in a different actual position than their static position in the HTML file, causing the hydration to fail.

```html
<!-- file.html -->
<a>
  outer
  <a>inner</a>
</a>

<!-- after rendering -->
<a>outer</a>
<a>inner</a>
```